packages/kbn-pm/.babelrc

```diff
@@ -9,6 +9,7 @@
   ],
   "plugins": [
     "@babel/proposal-class-properties",
-    "@babel/proposal-object-rest-spread"
+    "@babel/proposal-object-rest-spread",
+    "@babel/proposal-optional-chaining"
```

Comment:
is it unnecessary? nodejs supports optional chaining according to https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Reply:
according to https://node.green/ the next syntax is supported natively in 14.15.1:
- @babel/plugin-proposal-nullish-coalescing-operator
- @babel/plugin-proposal-object-rest-spread
- @babel/plugin-proposal-optional-chaining

Can we remove their usage for node env?

Reply:
See the "Note to reviewers" section in the PR description